products/confluence.md

```diff
@@ -16,8 +16,16 @@ auto:
 -   custom: true
 
 # release date: https://www.atlassian.com/software/confluence/download-archives
-# eol date: https://confluence.atlassian.com/support/atlassian-support-end-of-life-policy-201851003.html
+# lts/eol date: https://confluence.atlassian.com/support/atlassian-support-end-of-life-policy-201851003.html
 releases:
+-   releaseCycle: "8.7"
+    releaseDate: 2023-12-05
+    eol: 2025-12-06
```

Comment:
```suggestion
    eol: false # must be updated once the date appear on https://confluence.atlassian.com/support/atlassian-support-end-of-life-policy-201851003.html
```

8.7 is not an LTS (not indicated on the linked page, and the previous LTS, 8.5, was released not so long ago).